### PR TITLE
Fixing Issue #82

### PR DESCRIPTION
**Pull request by kartikeya23**
# Philip Paeps
	https://github.com/ppaeps
# Evgeny Potemkin
	https://www.facebook.com/evgeny.potemkin.9

Deployment Link: http://kartikeya23.github.io/2017.fossasia.org

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1195,7 +1196,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://mysqlserverteam.com/author/evgen/"><i class="icon fa fa-external-link"></i></a>
-
+                      <a target="default" href="https://www.facebook.com/evgeny.potemkin.9"><i class="icon social_facebook"></i></a>
```
Comment (abishekvashok) [inline index.html:1088]: Please properly intend the code

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -735,6 +735,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://wiki.freebsd.org/PhilipPaeps"><i class="icon fa fa-external-link"></i></a>
+                      <a target="default" href="https://github.com/ppaeps"><i class="icon social_github"></i></a>
```
Comment (nzec) [inline index.html]: Once again, indent properly.

File: index.html [hunk 3]
```diff
@@ -57,7 +57,7 @@
 								<a target="_self" href="http://fossasia.org"><img alt="FOSSASIA Summit" class="logo logo-light" src="img/fossasia-long.png">
 								<img alt="FOSSASIA" class="logo logo-dark" src="img/fossasia-dark.png"></a>
 							</div>
-							
+
```
Comment (harshit98) [inline index.html]: Remove extra space from here.

File: index.html [hunk 4]
```diff
@@ -137,7 +137,7 @@
                     </div>
                 </nav>
         </div>
-        
+
```
Comment (harshit98) [inline index.html]: Remove extra space from here too.

File: index.html [hunk 5]
```diff
@@ -202,18 +202,18 @@ <h1>Check out the schedule with 200+ sessions from speakers around the world.</h
             </section>
 
 			<a id="organizers" class="in-page-link"></a>
-			
+
 			<section class="speakers duplicatable-content">
-			
+
```
Comment (harshit98) [inline index.html]: Here as well.

File: index.html [hunk 6]
```diff
@@ -202,18 +202,18 @@ <h1>Check out the schedule with 200+ sessions from speakers around the world.</h
...
 					<div class="row">
 						<div class="col-sm-12">
 							<h1>Organizers</h1>
 						</div>
 					</div>
-					
+
 					<div class="row">
-				
+
```
Comment (harshit98) [inline index.html]: Extra space. Remove please.

**Comment by harshit98** [comment]
Hi @kartikeya23 Kindly do the suggested changes:
- Indent the code properly where it is necessary.
- Change your PR name to `Fixes #(issue-num): details of fix`.
- Remove extra spaces.

File: index.html (hunk 3 above)
Comment (kartikeya23) [reply index.html]: @harshit98 I had added that to visually segregate the two divs
Do you still want me to remove it?

File: index.html (hunk 2 above)
Comment (kartikeya23) [reply index.html]: It is already intended as it shows on my repo

**Comment by kartikeya23** [comment]
@abishekvashok @harshit98 @duttaditya18 
Did all the changes!
Please review

**Comment by nzec** [comment]
@kartikeya23 Resolve merge conflicts.

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@kartikeya23  Please resolve conflicts

**Comment by kartikeya23** [comment]
@ParthS007 @duttaditya18 Done!

File: index.html [hunk 7]
```diff
@@ -740,25 +665,21 @@ <h1>Featured Speakers</h1>
...
-
 						<div class="col-md-3 col-sm-6 speaker-column">
 							<div class="speaker">
 								<div class="image-holder">
 									<img class="background-image" alt="Philip Paeps" src="img/PhilipPaeps.jpg">
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://wiki.freebsd.org/PhilipPaeps"><i class="icon fa fa-external-link"></i></a>
-											<a target="default" href="https://github.com/ppaeps"><i class="icon fa fa-gitub"></i></a>
-											<a target="default" href="https://wiki.freebsd.org/PhilipPaeps"><i class="icon fa fa-external-link"></i></a>
```
Comment (harshit98) [inline index.html:753]: You're missing this link.

Comment (kartikeya23) [reply index.html:753]: Give me one sec

**Review by harshit98 — APPROVED** [review]

**Pull request merged**